src/cclib/io/moldenwriter.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""A writer for MOLDEN format files."""
+
+import os.path
+import ntpath
```

Comment:
Refactored the `title` extraction module to use only `os.path.basename` and `os.path.splitext`.